components/PlantCard/index.js

```diff
@@ -1,33 +1,35 @@
-import Image from "next/image"
-import styled from "styled-components"
+import Image from "next/image";
+import styled from "styled-components";
+import Link from "next/link";
 
-export default function PlantCard({ image, name, botanicalName }) {
-
-    return (
-        <article>
-            <StyledImageContainer>
-                <StyledImage
-                    src={image}
-                    alt={name} 
-                />
-            </StyledImageContainer>
-            <h2>{name}</h2>
-            <h3>{botanicalName}</h3>
-        </article>
-    )
+export default function PlantCard({ image, name, botanicalName, id }) {
+  return (
+    <article>
+      <StyledImageContainer>
+        <StyledImage
+          src={image}
+          alt={name}
+          fill={true}
+          style={{ objectFit: "cover" }}
+        ></StyledImage>
```

Comment:
Use a self closing tag please. And please do the styling in the styled component instead of an inline-style.

Reply:
Thank you for the feedback. We've made the tag self-closing and moved the `style={{ objectFit: "cover" }}` into the styled component, as you suggested. However, we prefer to keeop the `fill={true}` in the `<StyledImage />` itself as our app relies on it to render the images nicely.